ts/after-action-footer.tsx

```diff
@@ -13,11 +14,25 @@ export class AfterActionFooter extends React.Component<Props, State> {
     const donateLink = this.props.org.code == 'fp' ? "https://freepress.actionkit.com/donate/single/" :
                        this.props.org.code == 'dp' ? "https://secure.actblue.com/donate/nndayofaction?refcode=20170712-bftn" :
                        "https://donate.fightforthefuture.org/campaigns/bftnlanding/";
+
+    const protestLink = "https://actionnetwork.org/event_campaigns/show-up-and-speak-out-for-net-neutrality" + (this.props.zip ? "?zipcode=" + this.props.zip : "");
```

Comment:
thats all we have right now. at some point i was planning on bringing in the other fields too